funnel/templates/macros.html.jinja2

```diff
@@ -96,6 +96,13 @@
 {% macro project_header(project, project_save_form, class='', current_page='project') %}
   <div class="mui-container project-banner {% if class %}{{ class }}{% endif %}">
     <div class="grid">
+      {% if project.view_for('edit').is_available() and project.schedule_state.PUBLISHED and not project.schedule_start_at %}
```

Comment:
This should be a feature. Possible name: `schedule_publish_alert`

Reply:
Added `schedule_publish_alert` to project features

Reply:
It's not being used here?